lasagne/tests/layers/test_pool.py

```diff
@@ -469,6 +482,75 @@ def test_get_output_shape_for(self, input_shape, output_shape):
         #    pytest.skip()
 
 
+class TestUpscale2DLayer:
+    def scale_factor_test_sets():
+        for scale_factor in [2, 3]:
+                yield scale_factor
+
+    def input_layer(self, output_shape):
+        return Mock(get_output_shape=lambda: output_shape,
+                    output_shape=output_shape)
```

Comment:
Please remove the `get_output_shape` argument and only keep the `output_shape` one, I've gone rid of the last remnants in 25fd8c3! (No blame on you, you started your branch a day before that.)